mmpose/datasets/pipelines/bottom_up_transform.py

```diff
@@ -242,6 +242,90 @@ def __call__(self, joints):
         return visible_kpts
 
 
+class PAFGenerator:
+    """Generate part affinity fields.
+
+    Args:
+        output_res (int): Size of feature map.
+        limb_width (int): Limb width of part affinity fields.
+        skeleton (list[list]): connections of joints.
+    """
+
+    def __init__(self, output_res, limb_width, skeleton):
+        self.output_res = output_res
+        self.limb_width = limb_width
+        self.skeleton = skeleton
+
+    def _accumulate_paf_map(self, pafs, src, dst, count):
+        """Accumulate part affinity fields between two given joints.
+
+        Args:
+            pafs (np.ndarray[2xHxW]): paf maps (2 dimensions:x axis and
+                y axis) for a certain limb connection.
+            src (np.ndarray[2,]): coordinates of the source joint.
+            dst (np.ndarray[2,]): coordinates of the destination joint.
+            count (np.ndarray[HxW]): count map that preserves the number of
+                non-zero vectors at each point.
+
+        Note:
+            Args pafs and count are accumulated and returned.
```

Comment:
I don't understand this sentence.

Reply:
Could you please give any suggestion or example? In face, I do not really understand what kind of note should be given as you mentioned before. Thanks.

Reply:
Okay, remove this note then.

> as you mentioned before

Okay, the `count` was changed "in-place" in L301, which means that if you pass to this function an array, then the array will be changed after the function call. It is not safe.